applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -93,6 +93,12 @@ static void button_msg_sub_thread(void)
 					LOG_WRN("Failed to stop broadcast sink: %d", ret);
 				}
 			} else if (strm_state == STATE_PAUSED) {
+				ret = bt_mgmt_scan_start(0, 0, BT_MGMT_SCAN_TYPE_BROADCAST, NULL);
```

Comment:
Do we need to scan if the stream is just paused?

Reply:
Yes, since if user push button when in "pause state", then the sink needs to start the stream again, so sink needs to start scan and get sync with source.